x/ibc-hooks/contracts/crosschain-swaps/src/consts.rs

```diff
@@ -0,0 +1,9 @@
+// Msg Reply IDs
+pub const SWAP_REPLY_ID: u64 = 1u64;
+pub const FORWARD_REPLY_ID: u64 = 2u64;
```

Comment:
Should this be an enum?

Reply:
`u64` is required for reply id

This might work:
```rs
#[repr(u64)]
pub enum Reply {
  Swap = 1,
  Forward = 2,
}
```

Reply:
(This comment still stands)

Reply:
had to use a crate to get the value when passing it as a param, but it now is: 

```
use enum_repr::EnumRepr;

// Msg Reply IDs
#[EnumRepr(type = "u64")]
pub enum MsgReplyID {
    Swap = 1,
    Forward = 2,
}
```

Reply:
Cool cool!